### PR TITLE
tests: Move get GetSupportedCPUModels to libnode

### PR DESCRIPTION
**Pull request by RamLavi**
### What this PR does
This PR moves GetSupportedCPUModels to libnode where it is more relevant.

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [x] PR: The PR description is expressive enough and will help future contributors
- [x] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [x] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [x] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [x] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [x] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
NONE
```

### CODE REVIEW

File: tests/utils.go [hunk 1]
```diff
@@ -99,13 +99,13 @@ const (
 	DiskCustomHostPath     = "disk-custom-host-path"
 )
 
-func GetSupportedCPUModels(nodes k8sv1.NodeList) []string {
+func GetSupportedCPUModels(nodesList k8sv1.NodeList) []string {
```
Comment (codingben) [inline tests/utils.go]: I'd write it as `nodeList` which is similar to `k8sv1.NodeList`.

**Review by codingben — APPROVED** [review]
/lgtm

File: tests/BUILD.bazel [hunk 2]
```diff
@@ -13,7 +13,6 @@ go_library(
         "//pkg/certificates/triple/cert:go_default_library",
         "//pkg/pointer:go_default_library",
         "//pkg/util:go_default_library",
-        "//pkg/virt-controller/services:go_default_library",
```
Comment (AlonaKaplan) [inline tests/BUILD.bazel:16]: I don't understand how the PR caused this removal.

**Pull request merged**